draft-ietf-ppm-dap.md

```diff
@@ -2130,19 +2112,6 @@ for each input share in the job:
    private extension fields. If so, the Aggregator MUST mark the input share as
    invalid with error `invalid_message`.
 
-1. If the report pertains to a batch that was previously collected, then the
-   input share MUST be marked as invalid with error `batch_collected`.
-
-    * Implementation note: The Leader considers a batch to be collected once it
```

Comment:
I should put this back, but I'm not sure if it should go inline in `{#batch-buckets}` or in Operational Considerations.

Reply:
I put the implementation note unchanged at the bottom of {{batch-buckets}}.